api/v1alpha1/lvmcluster_webhook.go

```diff
@@ -164,6 +166,22 @@ func (l *LVMCluster) ValidateDelete() error {
 	return nil
 }
 
+func (l *LVMCluster) verifySingleDefaultDeviceClass() error {
+	if len(l.Spec.Storage.DeviceClasses) == 1 {
+		return nil
+	}
+	countDefault := 0
+	for _, deviceClass := range l.Spec.Storage.DeviceClasses {
+		if deviceClass.Default {
+			countDefault++
+		}
+	}
+	if countDefault != 1 {
```

Comment:
Is it possible for none of the DeviceClasses to be set Default (and therefore for countDefault to be 0)? If so, this would return a spurious error.

Reply:
I adjusted the function to return a different error message for each case.